public/assets/scripts/app.js

```diff
@@ -37,14 +37,8 @@ function resizeTextArea (elem) {
  * Resizes all textareas to match their content.
  */
 function resizeAll () {
-
     var fields = document.querySelectorAll(".copy-field");
-
-    // Iterate over each element and resize.
-    fields.map(function (field) {
-        resizeTextArea(field);
-    });
-
+    fields.map(resizeTextArea);
```

Comment:
oh, I just realized querySelectorAll return nodeList instead of array, and can't use map with it but [forEach](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach) instead. Can you change it to ```fields.forEach(resizeTextArea);``` ? Sorry for inconvenience.

Reply:
No worries. My bad for not testing it. I'll do this in the morning.